ui/selectors/selectors.js

```diff
@@ -595,21 +595,13 @@ export function getNumberOfTokens(state) {
 }
 
 export function getMetaMaskKeyrings(state) {
-  return state.metamask.keyrings.map((keyring, index) => ({
-    ...keyring,
-    metadata: state.metamask.keyringsMetadata?.[index] ?? {},
-  }));
+  return state.metamask.keyrings;
 }
 
-export const getMetaMaskHdKeyrings = createSelector(
```

Comment:
This `createSelector` used here is troublesome for unit tests, as the memoized value makes them interdependent

Reply:
Though I see that this pattern is used for several other selectors as well. Not sure why I started seeing this failing after these changes, but removing `createSelector` is still the only solution I found to be working